app/scripts/views/settings/settings-file-view.js

```diff
@@ -382,12 +396,27 @@ const SettingsFileView = Backbone.View.extend({
     blurMasterPass: function(e) {
         if (!e.target.value) {
             this.model.resetPassword();
-            e.target.value = PasswordGenerator.present(this.model.get('passwordLength'));
+            const password = PasswordGenerator.present(this.model.get('passwordLength'));
+            e.target.value = password;
             this.$el.find('.settings__file-master-pass-warning').hide();
+            this.$el.find('#settings__file-confirm-master-pass').val(password);
```

Comment:
Why? The confirmation field must be empty when the focus leaves the master password field, with a message that confirmation is required.

Reply:
Because of this `leave the field blank to use the old password`.